pages/api/_middleware.ts

```diff
@@ -1,10 +1,13 @@
 import type { NextApiRequest } from 'next'
 import { getToken } from 'next-auth/jwt'
+import getConfig from 'next/config';
 import { NextResponse } from 'next/server'
 
 import { error, info } from 'helpers/api/handle-log'
 
+const { serverRuntimeConfig } = getConfig();
```

Comment:
and then you'll be able to deconstruct it here without any `?` null coalescent needed

```ts
const {serverRuntimeConfig: { e2e } = getConfig();
```